packages/kernel-modules/kernel-modules/finalize.yaml

```diff
@@ -1,2 +1,2 @@
 install:
-- depmod -a
\ No newline at end of file
+- depmod -a {{ ( index .Values.labels "kernel_version" ) }}-mocaccino
```

Comment:
That's not really handy as requires to update the definition.yaml file each time. Can't we get the same info by looking /lib/modules content?

Reply:
I agree.
I propose something as you're proposal to @joostruis  in slack, he didn't seem convinced and words "too hackish" was used...
But I'm convinced that picking this from system is the best.

Reply:
@mudler done